### DISCUSSION TITLE
RPi zero ?

### DISCUSSION THREAD

**Discussion by Fish31520** [Nachtzuster/BirdNET-Pi#28] [Q&A] [answered]
Hi,

I have a RPI 4B which has been running Birdnet-Pi for the last 2 years or so.

I also have a RPI 0 WH in the back of a drawer. Do you think this fork could run on it ?

Thanks !

Jean

**Comment by Nachtzuster** [comment, marked as answer]
A RPI 0 W will not work.
A RPI 0 **2** W will, but it needs some tweaks before and after installing, follow: [RPi0W2-Installation-Guide](https://github.com/mcguirepr89/BirdNET-Pi/wiki/RPi0W2-Installation-Guide)

On top of that I would recommend enabling zram before installing:
`curl -s https://raw.githubusercontent.com/Nachtzuster/BirdNET-Pi/main/scripts/install_zram_service.sh | bash`

On final note: the RPi0W2-Installation-Guide recommends using 30s for `Recording Length`. But that is not really needed anymore.

**Reply by BAM-Team** [reply]
@Nachtzuster  What I can say from my testing, RPI02W is running much better on your Repo. Even the Species Stats Page is slow, but usable. Q: Could zram installation be done after installing, too?

**Reply by Nachtzuster** [reply]
Nice, thanks for the feedback!

Yes, you can do it afterwards. The recomendation to do it before, is because it should make the system more responsive and therefore easier to turn off unneeded services like livestream.
Since it is just a service, you can simply turn it off too, with `sudo systemctl disable zram.service`  & reboot.

**Reply by BAM-Team** [reply]
Great :-D Thank u so much.
